src/component-library/TokenBalance/TokenBalance.tsx

```diff
@@ -1,17 +1,19 @@
+import { formatUSD } from '@/common/utils/utils';
+
 import { TokenBalanceLabel, TokenBalanceValue, TokenBalanceWrapper } from './TokenBalance.style';
 
 interface TokenBalanceProps {
   tokenSymbol: string;
-  value: string;
-  valueInUSD: string;
+  value: number;
+  valueInUSD: number;
 }
 
 const TokenBalance = ({ tokenSymbol, value, valueInUSD }: TokenBalanceProps): JSX.Element => {
   return (
     <TokenBalanceWrapper>
       <TokenBalanceLabel>Balance:</TokenBalanceLabel>
       <TokenBalanceValue>
-        {value} {tokenSymbol} (${valueInUSD})
+        {value} {tokenSymbol} ({formatUSD(valueInUSD)})
```

Comment:
hi @crypto-engineer - I'd prefer to do the formatting outside of the component library if possible. The convention I'd like to use would be to avoid manipulating/changing a value in the component library wherever possible.

Reply:
You are right. I will update like so. I came to the same thought after creating this PR. :) Thank you!

Reply:
No worries :) thank you